build/0001-cloudflare.patch

```diff
@@ -0,0 +1,58 @@
+diff --git a/dh/x25519/BUILD.bazel b/dh/x25519/BUILD.bazel
```

Comment:
The cloudflare library doesn't compile correctly with the bazel build files it generations, so it needs this little patch to build.